libs/schemas/src/queries/thread.schemas.ts

```diff
@@ -91,3 +93,42 @@ export const GetBulkThreads = {
     threads: z.array(BulkThread),
   }),
 };
+
+export const GetThreads = {
+  input: PaginationParamsSchema.extend({
+    community_id: z.string(),
+    topic_id: PG_INT.optional(),
+    thread_id: PG_INT.optional(),
+    include_comments: z.coerce.boolean(),
+    include_user: z.coerce.boolean(),
+    include_reactions: z.coerce.boolean(),
+  }),
+  output: PaginatedResultSchema.extend({
+    results: Thread.array(),
+  }),
+};
+
+export const DEPRECATED_GetThreads = z.object({
```

Comment:
These schemas conflict with our naming convention but are used for the old API. I have prefixed them with DEPRECATED for now since I am unsure how to handle it.